tools/UpdatePaths/Scripts/68111_GAGs_clothesmate.txt

```diff
@@ -0,0 +1,11 @@
+#comment This replaces various clothing items with GAGs version https://github.com/tgstation/tgstation/pull/68111
```

Comment:
gonna be honest and say IDK how UpdatePaths scripts work but it'd be nice to know while manually mirroring this so I can fix all our uses of all the changed items 🥺